pages/1.11/administering-clusters/update-a-node/index.md

```diff
@@ -71,49 +71,58 @@ You can define a maintenance schedule to evacuate your tasks prior to changing a
 # Updating nodes by manually killing agents
 Draining nodes by using terminate signal, SIGUSR1, is easy to integrate with automation tools that can execute tasks on nodes in parallel, for example Ansible, Chef, and Puppet.
 
-1.  [SSH to the agent nodes](/1.11/administering-clusters/sshcluster/).
-1.  Stop the agents.
+1. Open a secure shell [SSH](/1.11/administering-clusters/sshcluster/) on the agent nodes.
 
-    -  **Private agent**
+1. Stop the agents by running the appropriate command.
+    -  For **private agents**, run:
 
        ```bash
        sudo sh -c 'systemctl kill -s SIGUSR1 dcos-mesos-slave && systemctl stop dcos-mesos-slave'
        ```
-    -  **Public agent**
+
+    - For **public agents**, run:
 
        ```bash
        ⁠⁠⁠⁠sudo sh -c 'systemctl kill -s SIGUSR1 dcos-mesos-slave-public && systemctl stop dcos-mesos-slave-public'
        ```
 
 1.  Perform your maintenance.
-1.  Add the nodes back to your cluster.
-    1.  Reload the systemd configuration.
 
-        ```bash
-        ﻿⁠⁠sudo systemctl daemon-reload
-        ```
+1.  Add the nodes back to your cluster by reloading the `systemd` configuration.
+
+    ```bash
+    ﻿⁠⁠sudo systemctl daemon-reload
+    ```
+    
+  If you are performing agent maintenance without changing agent attributes or resources, reloading the `systemd` configuration completes the node update and no other steps are needed.
 
-    1.  Remove the `latest` metadata pointer on the agent node:
+  If you are changing agent attributes or resources as part of updating the node, however, you should delete the `latest` symbolic link on the agent node to complete maintenance.
 
-        ```bash
```

Comment:
I wouldn't make "Removing agent metadata after maintenance" as a section heading. AFAICT, it only applies to the "rm" command.  The restart agent and checking status should be done irrespective of agent metadata change?